src/main/java/fr/lesprojetscagnottes/core/common/security/TokenProvider.java

```diff
@@ -89,12 +86,12 @@ public String generateToken(Authentication authentication) {
     }
 
     public String refreshToken(String token) {
-        final Date createdDate = clock.now();
+        final Date createdDate = new Date();
         final Date expirationDate = new Date(createdDate.getTime() + ACCESS_TOKEN_VALIDITY_SECONDS * 1000);
 
         SimpleDateFormat dt1 = new SimpleDateFormat("yyyyy-MM-dd");
-        LOGGER.debug("createdDate = " + dt1.format(createdDate));
-        LOGGER.debug("expirationDate = " + dt1.format(expirationDate));
+        log.debug("createdDate = " + dt1.format(createdDate));
```

Comment:
il manque la même correction que dans le fichier StringGenerator.java pour la date

Reply:
Ici, pour garder les choses simples, on est obligé de garder l'ancienne API car la library JWT manipule des objets java.util.Date

Reply:
Tu peux retransformer en Date ensuite si jamais : 
// LocalDateTime -> Date
LocalDateTime localDateTime = LocalDateTime.of(2020,2,20,21,46,31);
Date date2 = Date.from(localDateTime.atZone(ZoneId.systemDefault()).toInstant());

Reply:
Je prends note mais vu le cas d'usage je ne vois pas l'intérêt de passer par une conversion qui coute du CPU. Sans compter le fait que la méthode `refreshToken` est une des méthodes les plus appelées via l'API alors dans l'idéal il faut être assez optimisé 😄 
Autant fournir directement les objets Date attendus par la library JWT non ?

Reply:
Après niveau CPU la conversion ne doit pas coûter énormément je pense mais dans le doute...